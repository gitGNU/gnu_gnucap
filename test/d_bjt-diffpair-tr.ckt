difpair ckt - simple differential pair
* modified from Spice-3 examples

*.opt acct list node lvlcod=2 itermin=0 reltol=.01 abstol=1u vntol=.01
*>.opt rstray cstray method=euler
.option method=gear order=1
*.tf v(5) vin
*.dc vin -0.25 0.25 0.005

vin 1 0 sin(0 0.1 5meg) ac 1 dc 0
vcc 8 0 12
vee 9 0 -12
q1 4 2 6 0 qnl
q2 5 3 6 0 qnl
rs1 1 2 1k
rs2 3 0 1k
rc1 4 8 10k
rc2 5 8 10k
q3 6 7 9 0 qnl
q4 7 7 9 0 qnl
rbias 7 8 20k
*.model qnl npn(bf=80 rb=100 ccs=2pf tf=0.3ns tr=6ns cje=3pf cjc=2pf
*+   va=50)
.model qnl npn(bf=80 rb=100 va=50 tr=.4)

*>.print tran v(4) v(5)
*>.print op ccs(q*)
.op
.plot tran v(5)
*>.print tran qbc(q*)
*>.tran 5ns 500ns
*>.print tran cmu(q*)
*>.tran 5ns 500ns
*>.print tran vbc(q*)
*>.tran 5ns 500ns
*>.plot tran v(5)(2,10)
.tran 5ns 500ns

.plot ac vm(5) vp(5)
*>.plot ac vm(5)(-70,90) vp(5)(-70,90)
.ac dec 10 1 10ghz

.end
