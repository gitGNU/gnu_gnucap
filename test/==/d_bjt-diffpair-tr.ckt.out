difpair ckt - simple differential pair 
option method=gear order=1 
                         ^ ? need reverse, forward, or auto
option method=gear order=1 
                         ^ ? what's this?
#           ccs(Q1)    ccs(Q2)    ccs(Q3)    ccs(Q4)    ccs(qnl)  
 300.15     0.         0.         0.         0.         ??        
#Time       qbc(Q1)    qbc(Q2)    qbc(Q3)    qbc(Q4)    qbc(qnl)  
 0.        -2.095p    -2.095p    -4.1811p   -568.5E-18  ??        
 5.n       -2.068p    -2.11p     -4.1835p   -568.8E-18  ??        
 10.n      -2.0073p   -2.1598p   -4.1868p   -568.8E-18  ??        
 15.n      -1.9151p   -2.2446p   -4.1898p   -568.8E-18  ??        
 20.n      -1.8014p   -2.3523p   -4.1926p   -568.7E-18  ??        
 25.n      -1.6704p   -2.4785p   -4.1951p   -568.7E-18  ??        
 30.n      -1.5269p   -2.6185p   -4.1973p   -568.7E-18  ??        
 35.n      -1.3758p   -2.7671p   -4.1992p   -568.7E-18  ??        
 40.n      -1.2225p   -2.9191p   -4.2006p   -568.6E-18  ??        
 45.n      -1.0722p   -3.0694p   -4.2017p   -568.6E-18  ??        
 50.n      -930.01f   -3.2129p   -4.2022p   -568.5E-18  ??        
 55.n      -800.7f    -3.3447p   -4.2023p   -568.5E-18  ??        
 60.n      -688.59f   -3.4605p   -4.2019p   -568.4E-18  ??        
 65.n      -597.35f   -3.5565p   -4.201p    -568.4E-18  ??        
 70.n      -529.92f   -3.6297p   -4.1996p   -568.4E-18  ??        
 75.n      -488.42f   -3.6779p   -4.1977p   -568.3E-18  ??        
 80.n      -474.25f   -3.6993p   -4.1953p   -568.3E-18  ??        
 85.n      -488.06f   -3.6932p   -4.1926p   -568.2E-18  ??        
 90.n      -529.92f   -3.6592p   -4.1895p   -568.2E-18  ??        
 95.n      -599.31f   -3.5977p   -4.1862p   -568.2E-18  ??        
 100.n     -695.19f   -3.5094p   -4.1828p   -568.2E-18  ??        
 105.n     -816.f     -3.3957p   -4.1794p   -568.2E-18  ??        
 110.n     -959.62f   -3.2586p   -4.176p    -568.2E-18  ??        
 115.n     -1.1234p   -3.1005p   -4.1728p   -568.2E-18  ??        
 120.n     -1.3041p   -2.9247p   -4.1699p   -568.2E-18  ??        
 125.n     -1.4981p   -2.7347p   -4.1673p   -568.2E-18  ??        
 130.n     -1.701p    -2.5347p   -4.1651p   -568.3E-18  ??        
 135.n     -1.9086p   -2.3291p   -4.1634p   -568.3E-18  ??        
 140.n     -2.1159p   -2.1227p   -4.1622p   -568.4E-18  ??        
 145.n     -2.318p    -1.9203p   -4.1614p   -568.4E-18  ??        
 150.n     -2.5102p   -1.7267p   -4.1612p   -568.5E-18  ??        
 155.n     -2.6879p   -1.5468p   -4.1616p   -568.5E-18  ??        
 160.n     -2.8466p   -1.3847p   -4.1624p   -568.6E-18  ??        
 165.n     -2.9828p   -1.2443p   -4.1637p   -568.6E-18  ??        
 170.n     -3.0931p   -1.1289p   -4.1655p   -568.6E-18  ??        
 175.n     -3.1752p   -1.041p    -4.1675p   -568.7E-18  ??        
 180.n     -3.2274p   -982.39f   -4.17p     -568.7E-18  ??        
 185.n     -3.2488p   -954.16f   -4.1726p   -568.7E-18  ??        
 190.n     -3.2392p   -956.62f   -4.1754p   -568.7E-18  ??        
 195.n     -3.1991p   -989.39f   -4.1783p   -568.8E-18  ??        
 200.n     -3.1296p   -1.0515p   -4.1813p   -568.8E-18  ??        
 205.n     -3.0326p   -1.1413p   -4.1843p   -568.8E-18  ??        
 210.n     -2.9104p   -1.2565p   -4.1872p   -568.8E-18  ??        
 215.n     -2.766p    -1.3945p   -4.19p     -568.7E-18  ??        
 220.n     -2.603p    -1.5518p   -4.1926p   -568.7E-18  ??        
 225.n     -2.4253p   -1.7246p   -4.195p    -568.7E-18  ??        
 230.n     -2.2375p   -1.9086p   -4.197p    -568.7E-18  ??        
 235.n     -2.0442p   -2.0991p   -4.1987p   -568.6E-18  ??        
 240.n     -1.8504p   -2.2914p   -4.2001p   -568.6E-18  ??        
 245.n     -1.6612p   -2.4804p   -4.201p    -568.6E-18  ??        
 250.n     -1.4814p   -2.6612p   -4.2014p   -568.5E-18  ??        
 255.n     -1.3159p   -2.829p    -4.2014p   -568.5E-18  ??        
 260.n     -1.1689p   -2.9795p   -4.201p    -568.4E-18  ??        
 265.n     -1.0443p   -3.1088p   -4.2p      -568.4E-18  ??        
 270.n     -945.f     -3.2137p   -4.1986p   -568.4E-18  ??        
 275.n     -873.53f   -3.2916p   -4.1967p   -568.3E-18  ??        
 280.n     -831.43f   -3.3408p   -4.1944p   -568.3E-18  ??        
 285.n     -819.54f   -3.3603p   -4.1917p   -568.2E-18  ??        
 290.n     -837.95f   -3.3498p   -4.1888p   -568.2E-18  ??        
 295.n     -886.1f    -3.3095p   -4.1856p   -568.2E-18  ??        
 300.n     -962.8f    -3.2404p   -4.1823p   -568.2E-18  ??        
 305.n     -1.0663p   -3.1441p   -4.179p    -568.2E-18  ??        
 310.n     -1.1942p   -3.0228p   -4.1757p   -568.2E-18  ??        
 315.n     -1.3437p   -2.8792p   -4.1726p   -568.2E-18  ??        
 320.n     -1.5113p   -2.7167p   -4.1698p   -568.2E-18  ??        
 325.n     -1.6931p   -2.539p    -4.1672p   -568.2E-18  ??        
 330.n     -1.8848p   -2.3504p   -4.1651p   -568.3E-18  ??        
 335.n     -2.0817p   -2.1556p   -4.1634p   -568.3E-18  ??        
 340.n     -2.279p    -1.9593p   -4.1622p   -568.4E-18  ??        
 345.n     -2.4716p   -1.7666p   -4.1615p   -568.4E-18  ??        
 350.n     -2.6546p   -1.5823p   -4.1614p   -568.5E-18  ??        
 355.n     -2.8235p   -1.4111p   -4.1617p   -568.5E-18  ??        
 360.n     -2.9738p   -1.2575p   -4.1626p   -568.6E-18  ??        
 365.n     -3.1018p   -1.1253p   -4.1639p   -568.6E-18  ??        
 370.n     -3.2045p   -1.0177p   -4.1656p   -568.6E-18  ??        
 375.n     -3.2792p   -937.14f   -4.1677p   -568.7E-18  ??        
 380.n     -3.3246p   -885.48f   -4.1701p   -568.7E-18  ??        
 385.n     -3.3395p   -863.72f   -4.1728p   -568.7E-18  ??        
 390.n     -3.324p    -872.14f   -4.1756p   -568.7E-18  ??        
 395.n     -3.2783p   -910.41f   -4.1785p   -568.8E-18  ??        
 400.n     -3.2038p   -977.53f   -4.1814p   -568.8E-18  ??        
 405.n     -3.1022p   -1.0719p   -4.1844p   -568.8E-18  ??        
 410.n     -2.9758p   -1.1914p   -4.1873p   -568.8E-18  ??        
 415.n     -2.8275p   -1.3332p   -4.19p     -568.7E-18  ??        
 420.n     -2.6608p   -1.4941p   -4.1926p   -568.7E-18  ??        
 425.n     -2.4798p   -1.6702p   -4.195p    -568.7E-18  ??        
 430.n     -2.2888p   -1.8574p   -4.197p    -568.7E-18  ??        
 435.n     -2.0925p   -2.0509p   -4.1987p   -568.6E-18  ??        
 440.n     -1.8959p   -2.246p    -4.2p      -568.6E-18  ??        
 445.n     -1.7039p   -2.4377p   -4.201p    -568.6E-18  ??        
 450.n     -1.5216p   -2.621p    -4.2014p   -568.5E-18  ??        
 455.n     -1.3536p   -2.7913p   -4.2014p   -568.5E-18  ??        
 460.n     -1.2042p   -2.9442p   -4.2009p   -568.4E-18  ??        
 465.n     -1.0772p   -3.0758p   -4.2p      -568.4E-18  ??        
 470.n     -975.76f   -3.1828p   -4.1985p   -568.4E-18  ??        
 475.n     -902.21f   -3.2629p   -4.1966p   -568.3E-18  ??        
 480.n     -858.17f   -3.314p    -4.1943p   -568.3E-18  ??        
 485.n     -844.45f   -3.3353p   -4.1917p   -568.2E-18  ??        
 490.n     -861.18f   -3.3264p   -4.1887p   -568.2E-18  ??        
 495.n     -907.79f   -3.2877p   -4.1856p   -568.2E-18  ??        
 500.n     -983.1f    -3.22p     -4.1823p   -568.2E-18  ??        
#Time       cmu(Q1)    cmu(Q2)    cmu(Q3)    cmu(Q4)    cmu(qnl)  
 0.         400.01f    400.01f    400.f      401.46f    ??        
 5.n        400.01f    400.01f    400.f      401.46f    ??        
 10.n       400.01f    400.01f    400.f      401.46f    ??        
 15.n       400.01f    400.01f    400.f      401.46f    ??        
 20.n       400.01f    400.01f    400.f      401.46f    ??        
 25.n       400.01f    400.01f    400.f      401.46f    ??        
 30.n       400.01f    400.01f    400.f      401.46f    ??        
 35.n       400.01f    400.01f    400.f      401.46f    ??        
 40.n       400.01f    400.01f    400.f      401.46f    ??        
 45.n       400.01f    400.01f    400.f      401.46f    ??        
 50.n       400.02f    400.f      400.f      401.46f    ??        
 55.n       400.02f    400.f      400.f      401.46f    ??        
 60.n       400.02f    400.f      400.f      401.46f    ??        
 65.n       400.03f    400.f      400.f      401.46f    ??        
 70.n       400.03f    400.f      400.f      401.46f    ??        
 75.n       400.03f    400.f      400.f      401.46f    ??        
 80.n       400.03f    400.f      400.f      401.46f    ??        
 85.n       400.03f    400.f      400.f      401.46f    ??        
 90.n       400.03f    400.f      400.f      401.46f    ??        
 95.n       400.03f    400.f      400.f      401.46f    ??        
 100.n      400.02f    400.f      400.f      401.46f    ??        
 105.n      400.02f    400.f      400.f      401.46f    ??        
 110.n      400.02f    400.f      400.f      401.46f    ??        
 115.n      400.01f    400.01f    400.f      401.46f    ??        
 120.n      400.01f    400.01f    400.f      401.46f    ??        
 125.n      400.01f    400.01f    400.f      401.46f    ??        
 130.n      400.01f    400.01f    400.f      401.46f    ??        
 135.n      400.01f    400.01f    400.f      401.46f    ??        
 140.n      400.01f    400.01f    400.f      401.46f    ??        
 145.n      400.01f    400.01f    400.f      401.46f    ??        
 150.n      400.01f    400.01f    400.f      401.46f    ??        
 155.n      400.01f    400.01f    400.f      401.46f    ??        
 160.n      400.01f    400.01f    400.f      401.46f    ??        
 165.n      400.01f    400.01f    400.f      401.46f    ??        
 170.n      400.01f    400.01f    400.f      401.46f    ??        
 175.n      400.01f    400.02f    400.f      401.46f    ??        
 180.n      400.f      400.02f    400.f      401.46f    ??        
 185.n      400.f      400.02f    400.f      401.46f    ??        
 190.n      400.f      400.02f    400.f      401.46f    ??        
 195.n      400.01f    400.02f    400.f      401.46f    ??        
 200.n      400.01f    400.02f    400.f      401.46f    ??        
 205.n      400.01f    400.01f    400.f      401.46f    ??        
 210.n      400.01f    400.01f    400.f      401.46f    ??        
 215.n      400.01f    400.01f    400.f      401.46f    ??        
 220.n      400.01f    400.01f    400.f      401.46f    ??        
 225.n      400.01f    400.01f    400.f      401.46f    ??        
 230.n      400.01f    400.01f    400.f      401.46f    ??        
 235.n      400.01f    400.01f    400.f      401.46f    ??        
 240.n      400.01f    400.01f    400.f      401.46f    ??        
 245.n      400.01f    400.01f    400.f      401.46f    ??        
 250.n      400.01f    400.01f    400.f      401.46f    ??        
 255.n      400.01f    400.01f    400.f      401.46f    ??        
 260.n      400.01f    400.01f    400.f      401.46f    ??        
 265.n      400.02f    400.01f    400.f      401.46f    ??        
 270.n      400.02f    400.f      400.f      401.46f    ??        
 275.n      400.02f    400.f      400.f      401.46f    ??        
 280.n      400.02f    400.f      400.f      401.46f    ??        
 285.n      400.02f    400.f      400.f      401.46f    ??        
 290.n      400.02f    400.f      400.f      401.46f    ??        
 295.n      400.02f    400.f      400.f      401.46f    ??        
 300.n      400.02f    400.f      400.f      401.46f    ??        
 305.n      400.02f    400.01f    400.f      401.46f    ??        
 310.n      400.01f    400.01f    400.f      401.46f    ??        
 315.n      400.01f    400.01f    400.f      401.46f    ??        
 320.n      400.01f    400.01f    400.f      401.46f    ??        
 325.n      400.01f    400.01f    400.f      401.46f    ??        
 330.n      400.01f    400.01f    400.f      401.46f    ??        
 335.n      400.01f    400.01f    400.f      401.46f    ??        
 340.n      400.01f    400.01f    400.f      401.46f    ??        
 345.n      400.01f    400.01f    400.f      401.46f    ??        
 350.n      400.01f    400.01f    400.f      401.46f    ??        
 355.n      400.01f    400.01f    400.f      401.46f    ??        
 360.n      400.01f    400.01f    400.f      401.46f    ??        
 365.n      400.01f    400.01f    400.f      401.46f    ??        
 370.n      400.f      400.02f    400.f      401.46f    ??        
 375.n      400.f      400.02f    400.f      401.46f    ??        
 380.n      400.f      400.02f    400.f      401.46f    ??        
 385.n      400.f      400.02f    400.f      401.46f    ??        
 390.n      400.f      400.02f    400.f      401.46f    ??        
 395.n      400.f      400.02f    400.f      401.46f    ??        
 400.n      400.f      400.02f    400.f      401.46f    ??        
 405.n      400.01f    400.01f    400.f      401.46f    ??        
 410.n      400.01f    400.01f    400.f      401.46f    ??        
 415.n      400.01f    400.01f    400.f      401.46f    ??        
 420.n      400.01f    400.01f    400.f      401.46f    ??        
 425.n      400.01f    400.01f    400.f      401.46f    ??        
 430.n      400.01f    400.01f    400.f      401.46f    ??        
 435.n      400.01f    400.01f    400.f      401.46f    ??        
 440.n      400.01f    400.01f    400.f      401.46f    ??        
 445.n      400.01f    400.01f    400.f      401.46f    ??        
 450.n      400.01f    400.01f    400.f      401.46f    ??        
 455.n      400.01f    400.01f    400.f      401.46f    ??        
 460.n      400.01f    400.01f    400.f      401.46f    ??        
 465.n      400.01f    400.01f    400.f      401.46f    ??        
 470.n      400.02f    400.01f    400.f      401.46f    ??        
 475.n      400.02f    400.f      400.f      401.46f    ??        
 480.n      400.02f    400.f      400.f      401.46f    ??        
 485.n      400.02f    400.f      400.f      401.46f    ??        
 490.n      400.02f    400.f      400.f      401.46f    ??        
 495.n      400.02f    400.f      400.f      401.46f    ??        
 500.n      400.02f    400.f      400.f      401.46f    ??        
#Time       vbc(Q1)    vbc(Q2)    vbc(Q3)    vbc(Q4)    vbc(qnl)  
 0.        -5.2368    -5.2368    -10.451     0.         ??        
 5.n       -5.1682    -5.2745    -10.457     0.         ??        
 10.n      -5.0161    -5.3998    -10.465     0.         ??        
 15.n      -4.7849    -5.6123    -10.473     0.         ??        
 20.n      -4.5002    -5.8822    -10.48      0.         ??        
 25.n      -4.1724    -6.1982    -10.486     0.         ??        
 30.n      -3.8132    -6.5483    -10.492     0.         ??        
 35.n      -3.4354    -6.92      -10.496     0.         ??        
 40.n      -3.0521    -7.3002    -10.5       0.         ??        
 45.n      -2.6763    -7.676     -10.503     0.         ??        
 50.n      -2.3209    -8.0346    -10.504     0.         ??        
 55.n      -1.9978    -8.3639    -10.504     0.         ??        
 60.n      -1.7179    -8.6531    -10.503     0.         ??        
 65.n      -1.4902    -8.8928    -10.501     0.         ??        
 70.n      -1.322     -9.0754    -10.497     0.         ??        
 75.n      -1.2188    -9.1952    -10.493     0.         ??        
 80.n      -1.1839    -9.2483    -10.487     0.         ??        
 85.n      -1.219     -9.2325    -10.48      0.         ??        
 90.n      -1.3242    -9.147     -10.472     0.         ??        
 95.n      -1.4983    -8.9926    -10.464     0.         ??        
 100.n     -1.7386    -8.7712    -10.455     0.         ??        
 105.n     -2.0411    -8.4864    -10.447     0.         ??        
 110.n     -2.4007    -8.1431    -10.438     0.         ??        
 115.n     -2.8106    -7.7476    -10.43      0.         ??        
 120.n     -3.2628    -7.3076    -10.423     0.         ??        
 125.n     -3.748     -6.8323    -10.417     0.         ??        
 130.n     -4.2557    -6.332     -10.411     0.         ??        
 135.n     -4.7747    -5.8178    -10.407     0.         ??        
 140.n     -5.293     -5.3017    -10.404     0.         ??        
 145.n     -5.7983    -4.7957    -10.402     0.         ??        
 150.n     -6.2788    -4.312     -10.402     0.         ??        
 155.n     -6.7226    -3.8622    -10.402     0.         ??        
 160.n     -7.1192    -3.4572    -10.405     0.         ??        
 165.n     -7.4591    -3.1067    -10.408     0.         ??        
 170.n     -7.7344    -2.8186    -10.412     0.         ??        
 175.n     -7.9392    -2.5994    -10.417     0.         ??        
 180.n     -8.0691    -2.4535    -10.423     0.         ??        
 185.n     -8.122     -2.3836    -10.43      0.         ??        
 190.n     -8.0973    -2.3903    -10.437     0.         ??        
 195.n     -7.9963    -2.4729    -10.444     0.         ??        
 200.n     -7.822     -2.6288    -10.452     0.         ??        
 205.n     -7.5788    -2.8538    -10.459     0.         ??        
 210.n     -7.2728    -3.1425    -10.467     0.         ??        
 215.n     -6.9114    -3.4879    -10.474     0.         ??        
 220.n     -6.5035    -3.8816    -10.48      0.         ??        
 225.n     -6.059     -4.3139    -10.486     0.         ??        
 230.n     -5.5891    -4.7742    -10.491     0.         ??        
 235.n     -5.1057    -5.2508    -10.495     0.         ??        
 240.n     -4.6211    -5.7316    -10.499     0.         ??        
 245.n     -4.1481    -6.2041    -10.501     0.         ??        
 250.n     -3.6989    -6.6559    -10.502     0.         ??        
 255.n     -3.2853    -7.0752    -10.502     0.         ??        
 260.n     -2.9182    -7.4512    -10.501     0.         ??        
 265.n     -2.6069    -7.774     -10.498     0.         ??        
 270.n     -2.3592    -8.0358    -10.495     0.         ??        
 275.n     -2.1811    -8.2301    -10.49      0.         ??        
 280.n     -2.0764    -8.3526    -10.484     0.         ??        
 285.n     -2.0473    -8.4008    -10.478     0.         ??        
 290.n     -2.094     -8.3737    -10.47      0.         ??        
 295.n     -2.2149    -8.2723    -10.462     0.         ??        
 300.n     -2.4073    -8.099     -10.454     0.         ??        
 305.n     -2.6666    -7.8578    -10.446     0.         ??        
 310.n     -2.987     -7.554     -10.438     0.         ??        
 315.n     -3.3612    -7.1945    -10.43      0.         ??        
 320.n     -3.7806    -6.7877    -10.423     0.         ??        
 325.n     -4.2355    -6.3431    -10.417     0.         ??        
 330.n     -4.715     -5.8714    -10.411     0.         ??        
 335.n     -5.2074    -5.3841    -10.407     0.         ??        
 340.n     -5.7006    -4.8933    -10.404     0.         ??        
 345.n     -6.1821    -4.4115    -10.402     0.         ??        
 350.n     -6.6396    -3.9509    -10.402     0.         ??        
 355.n     -7.0615    -3.5233    -10.403     0.         ??        
 360.n     -7.437     -3.1396    -10.405     0.         ??        
 365.n     -7.7566    -2.8094    -10.408     0.         ??        
 370.n     -8.0127    -2.5408    -10.413     0.         ??        
 375.n     -8.1991    -2.34      -10.418     0.         ??        
 380.n     -8.3119    -2.2114    -10.424     0.         ??        
 385.n     -8.3486    -2.1576    -10.43      0.         ??        
 390.n     -8.3091    -2.1793    -10.437     0.         ??        
 395.n     -8.1944    -2.2755    -10.445     0.         ??        
 400.n     -8.0076    -2.444     -10.452     0.         ??        
 405.n     -7.7528    -2.6805    -10.46      0.         ??        
 410.n     -7.4362    -2.9798    -10.467     0.         ??        
 415.n     -7.0651    -3.3348    -10.474     0.         ??        
 420.n     -6.648     -3.7375    -10.48      0.         ??        
 425.n     -6.1951    -4.1781    -10.486     0.         ??        
 430.n     -5.7173    -4.6463    -10.491     0.         ??        
 435.n     -5.2264    -5.1303    -10.495     0.         ??        
 440.n     -4.7348    -5.6181    -10.499     0.         ??        
 445.n     -4.255     -6.0973    -10.501     0.         ??        
 450.n     -3.7993    -6.5555    -10.502     0.         ??        
 455.n     -3.3795    -6.9811    -10.502     0.         ??        
 460.n     -3.0063    -7.363     -10.501     0.         ??        
 465.n     -2.6893    -7.6916    -10.498     0.         ??        
 470.n     -2.4361    -7.9588    -10.495     0.         ??        
 475.n     -2.2528    -8.1583    -10.49      0.         ??        
 480.n     -2.1432    -8.2856    -10.484     0.         ??        
 485.n     -2.1096    -8.3383    -10.478     0.         ??        
 490.n     -2.152     -8.3154    -10.47      0.         ??        
 495.n     -2.2692    -8.2179    -10.462     0.         ??        
 500.n     -2.458     -8.0481    -10.454     0.         ??        
v(5)   2.                4.               6.                8.              10.
        +-----------------+----------------+-----------------+----------------+
 0.     |                 .         *      .                 .                |
 5.n    |                 .          *     .                 .                |
 10.n   |                 .           *    .                 .                |
 15.n   |                 .             *  .                 .                |
 20.n   |                 .               *.                 .                |
 25.n   |                 .                . *               .                |
 30.n   |                 .                .    *            .                |
 35.n   |                 .                .       *         .                |
 40.n   |                 .                .           *     .                |
 45.n   |                 .                .              *  .                |
 50.n   |                 .                .                 *                |
 55.n   |                 .                .                 .  *             |
 60.n   |                 .                .                 .    *           |
 65.n   |                 .                .                 .      *         |
 70.n   |                 .                .                 .        *       |
 75.n   |                 .                .                 .         *      |
 80.n   |                 .                .                 .         *      |
 85.n   |                 .                .                 .         *      |
 90.n   |                 .                .                 .        *       |
 95.n   |                 .                .                 .       *        |
 100.n  |                 .                .                 .     *          |
 105.n  |                 .                .                 .   *            |
 110.n  |                 .                .                 *                |
 115.n  |                 .                .              *  .                |
 120.n  |                 .                .          *      .                |
 125.n  |                 .                .      *          .                |
 130.n  |                 .                . *               .                |
 135.n  |                 .              * .                 .                |
 140.n  |                 .         *      .                 .                |
 145.n  |                 .     *          .                 .                |
 150.n  |                 . *              .                 .                |
 155.n  |               * .                .                 .                |
 160.n  |           *     .                .                 .                |
 165.n  |        *        .                .                 .                |
 170.n  |      *          .                .                 .                |
 175.n  |    *            .                .                 .                |
 180.n  |   *             .                .                 .                |
 185.n  |  *              .                .                 .                |
 190.n  |  *              .                .                 .                |
 195.n  |   *             .                .                 .                |
 200.n  |     *           .                .                 .                |
 205.n  |       *         .                .                 .                |
 210.n  |         *       .                .                 .                |
 215.n  |            *    .                .                 .                |
 220.n  |                *.                .                 .                |
 225.n  |                 . *              .                 .                |
 230.n  |                 .      *         .                 .                |
 235.n  |                 .          *     .                 .                |
 240.n  |                 .              * .                 .                |
 245.n  |                 .                . *               .                |
 250.n  |                 .                .     *           .                |
 255.n  |                 .                .         *       .                |
 260.n  |                 .                .            *    .                |
 265.n  |                 .                .               * .                |
 270.n  |                 .                .                 *                |
 275.n  |                 .                .                 . *              |
 280.n  |                 .                .                 .  *             |
 285.n  |                 .                .                 .  *             |
 290.n  |                 .                .                 .  *             |
 295.n  |                 .                .                 . *              |
 300.n  |                 .                .                 *                |
 305.n  |                 .                .               * .                |
 310.n  |                 .                .            *    .                |
 315.n  |                 .                .         *       .                |
 320.n  |                 .                .      *          .                |
 325.n  |                 .                .  *              .                |
 330.n  |                 .              * .                 .                |
 335.n  |                 .          *     .                 .                |
 340.n  |                 .      *         .                 .                |
 345.n  |                 .  *             .                 .                |
 350.n  |                *.                .                 .                |
 355.n  |            *    .                .                 .                |
 360.n  |         *       .                .                 .                |
 365.n  |      *          .                .                 .                |
 370.n  |   *             .                .                 .                |
 375.n  |  *              .                .                 .                |
 380.n  | *               .                .                 .                |
 385.n  |*                .                .                 .                |
 390.n  |*                .                .                 .                |
 395.n  | *               .                .                 .                |
 400.n  |   *             .                .                 .                |
 405.n  |     *           .                .                 .                |
 410.n  |        *        .                .                 .                |
 415.n  |           *     .                .                 .                |
 420.n  |              *  .                .                 .                |
 425.n  |                 .*               .                 .                |
 430.n  |                 .    *           .                 .                |
 435.n  |                 .         *      .                 .                |
 440.n  |                 .             *  .                 .                |
 445.n  |                 .                .*                .                |
 450.n  |                 .                .    *            .                |
 455.n  |                 .                .        *        .                |
 460.n  |                 .                .           *     .                |
 465.n  |                 .                .              *  .                |
 470.n  |                 .                .                *.                |
 475.n  |                 .                .                 .*               |
 480.n  |                 .                .                 . *              |
 485.n  |                 .                .                 . *              |
 490.n  |                 .                .                 . *              |
 495.n  |                 .                .                 .*               |
 500.n  |                 .                .                 *                |
        +-----------------+----------------+-----------------+----------------+
vm(5) -70.              -30.              10.               50.             90.
vp(5) -70.              -30.              10.               50.             90.
        +-----------------+----------------+-----------------+----------------+
 1.     |                 .            +   .                 .           *    |
 1.2589 |                 .            +   .                 .           *    |
 1.5849 |                 .            +   .                 .           *    |
 1.9953 |                 .            +   .                 .           *    |
 2.5119 |                 .            +   .                 .           *    |
 3.1623 |                 .            +   .                 .           *    |
 3.9811 |                 .            +   .                 .           *    |
 5.0119 |                 .            +   .                 .           *    |
 6.3096 |                 .            +   .                 .           *    |
 7.9433 |                 .            +   .                 .           *    |
 10.    |                 .            +   .                 .           *    |
 12.589 |                 .            +   .                 .           *    |
 15.849 |                 .            +   .                 .           *    |
 19.953 |                 .            +   .                 .           *    |
 25.119 |                 .            +   .                 .           *    |
 31.623 |                 .            +   .                 .           *    |
 39.811 |                 .            +   .                 .           *    |
 50.119 |                 .            +   .                 .           *    |
 63.096 |                 .            +   .                 .           *    |
 79.433 |                 .            +   .                 .           *    |
 100.   |                 .            +   .                 .           *    |
 125.89 |                 .            +   .                 .           *    |
 158.49 |                 .            +   .                 .           *    |
 199.53 |                 .            +   .                 .           *    |
 251.19 |                 .            +   .                 .           *    |
 316.23 |                 .            +   .                 .           *    |
 398.11 |                 .            +   .                 .           *    |
 501.19 |                 .            +   .                 .           *    |
 630.96 |                 .            +   .                 .           *    |
 794.33 |                 .            +   .                 .           *    |
 1.K    |                 .            +   .                 .           *    |
 1.2589K|                 .            +   .                 .           *    |
 1.5849K|                 .            +   .                 .           *    |
 1.9953K|                 .            +   .                 .           *    |
 2.5119K|                 .            +   .                 .           *    |
 3.1623K|                 .            +   .                 .           *    |
 3.9811K|                 .            +   .                 .           *    |
 5.0119K|                 .            +   .                 .           *    |
 6.3096K|                 .            +   .                 .           *    |
 7.9433K|                 .            +   .                 .           *    |
 10.K   |                 .            +   .                 .           *    |
 12.589K|                 .           +    .                 .           *    |
 15.849K|                 .           +    .                 .           *    |
 19.953K|                 .           +    .                 .           *    |
 25.119K|                 .           +    .                 .           *    |
 31.623K|                 .           +    .                 .           *    |
 39.811K|                 .           +    .                 .           *    |
 50.119K|                 .           +    .                 .           *    |
 63.096K|                 .           +    .                 .           *    |
 79.433K|                 .           +    .                 .           *    |
 100.K  |                 .          +     .                 .           *    |
 125.89K|                 .          +     .                 .           *    |
 158.49K|                 .          +     .                 .           *    |
 199.53K|                 .         +      .                 .           *    |
 251.19K|                 .         +      .                 .           *    |
 316.23K|                 .        +       .                 .          *     |
 398.11K|                 .       +        .                 .          *     |
 501.19K|                 .      +         .                 .          *     |
 630.96K|                 .     +          .                 .         *      |
 794.33K|                 .   +            .                 .         *      |
 1.Meg  |                 . +              .                 .        *       |
 1.2589M|                 +                .                 .      *         |
 1.5849M|              +  .                .                 .    *           |
 1.9953M|           +     .                .                 .  *             |
 2.5119M|        +        .                .                 *                |
 3.1623M|     +           .                .              *  .                |
 3.9811M|   +             .                .           *     .                |
 5.0119M|+                .                .        *        .                |
 6.3096M+                 .                .      *          .                |
 7.9433M+                 .                .    *            .                |
 10.Meg +                 .                .  *              .                |
 12.589M+                 .                . *               .                |
 15.849M+                 .                *                 .                |
 19.953M+                 .                *                 .                |
 25.119M+                 .               *.                 .                |
 31.623M+                 .              * .                 .                |
 39.811M+                 .              * .                 .                |
 50.119M+                 .             *  .                 .                |
 63.096M+                 .             *  .                 .                |
 79.433M+                 .             *  .                 .                |
 100.Meg+                 .            *   .                 .                |
 125.89M+                 .            *   .                 .                |
 158.49M+                 .            *   .                 .                |
 199.53M+                 .            *   .                 .                |
 251.19M+                 .            *   .                 .                |
 316.23M+                 .            *   .                 .                |
 398.11M+                 .            *   .                 .                |
 501.19M+                 .            *   .                 .                |
 630.96M+                 .            *   .                 .                |
 794.33M+                 .            *   .                 .                |
 1.G    +                 .            *   .                 .                |
 1.2589G+                 .            *   .                 .                |
 1.5849G+                 .            *   .                 .                |
 1.9953G|+                .            *   .                 .                |
 2.5119G|   +             .            *   .                 .                |
 3.1623G|       +         .            *   .                 .                |
 3.9811G|          +      .            *   .                 .                |
 5.0119G|             +   .            *   .                 .                |
 6.3096G|                +.            *   .                 .                |
 7.9433G|                 .+           *   .                 .                |
 10.G   |                 .   +        *   .                 .                |
        +-----------------+----------------+-----------------+----------------+
